From: Jeff Mahoney <jeffm@suse.com>
Subject: [PATCH] velociraptor: remove ignore tag to allow vendoring of mage

The ignore tag in make.go means it won't be properly vendored.

---
 make.go |    2 --
 1 file changed, 2 deletions(-)

diff --git a/make.go b/make.go
index 28b3e90..8fad8b9 100644
--- a/make.go
+++ b/make.go
@@ -1,5 +1,3 @@
-// +build ignore
-
 /*
    Velociraptor - Dig Deeper
    Copyright (C) 2019-2022 Rapid7 Inc.
